modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc
+:_content-type: PROCEDURE
+
+[id="op-running-as-root-user-namespace_{context}"]
+= Running buildah as root in a user namespace
+
+If you are using {pipelines-shortname} to run pods in the user namespace, you must pass the annotations that the CRI-O container engine expects. You can add these annotations for all users, or for a specific namespace.
+
+.Procedure
+
+. Export the `buildah` cluster task as a task to the project's namepsace.
++
+[source,terminal]
+----
+$ tkn task create --from=buildah
+----
+
+. Edit the `buildah` task within the project's namespace:
++
+[source,terminal]
+----
+$ oc edit task buildah
+----
+
+. Add annotations to the task:
++
+[source,yaml]
+----
+...
+annotations:
+  io.kubernetes.cri-o.userns-mode: "auto"
+  io.openshift.builder: "true"
+...
+----
+
+. Optional: Cluster administrators can verify that the `buildah` task's container is running as root in the user-defined namespace. 
+
+.. Get the name of the node where the buidah container is running:
++
+[source,terminal]
+----
+$ oc get pod -o wide
+----
+
+.. Run a debugging pod on the node where the `buildah` pod is running:
++
+[source,terminal]
+----
+$ oc debug nodes/<node-name>
+----
+
+.. Change the root directory for the current running processes:
++
+[source,terminal]
+----
+$ chroot /host
+----
+
+.. List the user namespaces:
++
+[source,terminal]
+----
+$ lsns -t user
+----
+
+[NOTE]
+====
+* Although {pipelines-shortname} runs your `buildah` container as a root user, CRI-O runs the container as a user in the host namespace.
+
+* When you run a command in a user namespace, your container runs as root (user id 0) but has user privileges on the host. {product-title} uses CRI-O as its container runtime interface. CRI-O has a workload feature to run a pod in the user namespace.
+====
+
+You can use the following `buildah` task configuration for reference:
```

Comment:
```suggestion
```
Consider removing this intro sentence for minimalism. It's not adding any new content that's not already implied by the `.Example` heading -- that is, because you have the heading `.Example: Modified task`, the user knows the sample is meant as a reference.